player/loadfile.c

```diff
@@ -852,8 +852,9 @@ int mp_add_external_file(struct MPContext *mpctx, char *filename,
 
     mp_core_unlock(mpctx);
 
+    char *path = mp_get_user_path(NULL, mpctx->global, filename);
     struct demuxer *demuxer =
-        demux_open_url(filename, &params, cancel, mpctx->global);
+        demux_open_url(path, &params, cancel, mpctx->global);
```

Comment:
Why not `talloc_free(path);` immediately after this line?

Reply:
I wasn't sure if freeing `path` would cause some issue with any of the demuxer stuff trying to use the url, but it looks like stream does a copy so I guess it's probably OK?

Reply:
Should be fine and if not it is on stream.c to fix.